src/lib/util.js

```diff
@@ -56,6 +56,19 @@ const filterUndefined = (fields) => {
   return fields
 }
 
+/**
+ * Method to provide object clonning
+ *
+ * TODO:
+ *  Implement a better deep copy method
+ *
+ * @param value
+ * @returns {any}
+ */
+const clone = (value) => {
```

Comment:
i think the name of this method is confusing because it actually creates a JSON object from a string, so not clone more transform

Reply:
It is a clone, it is however only a shallowClone since its only copying properties and not functions. Do you not agree?